e2e/editor.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { test, expect, type Page } from '@playwright/test';
+
+let page: Page;
```

Comment:
```suggestion
import { test, expect } from '@playwright/test';
```

We don't need a global page because...